src/_locales/en_US/messages.json

```diff
@@ -173,6 +173,14 @@
         "message": "Learn in Private/Incognito windows",
         "description": "Checkbox label on the general settings page"
     },
+    "options_community_learning_setting": {
+        "message": "Enable community learning and share data about trackers",
+        "description": "Checkbox label on the general settings page"
+    },
+    "options_community_learning_warning": {
```

Comment:
Should the link that explains what community learning is (and the decision making that led to it) be included in this message or the one above?